src/mock-typeorm.ts

```diff
@@ -61,12 +61,15 @@ export class MockTypeORM {
           const totalMockItemsFoundInMethod = Object.keys(mockMethod).length;
           mockMethod[totalMockItemsFoundInMethod] = mockData;
         } else {
-          self.mocks[repositoryName] = { [method]: { 0: mockData } };
+          //self.mocks[repositoryName] = { [method]: { 0: mockData } };
+          self.mocks[repositoryName][method] = { 0: mockData };
 
           // initialize mock history with empty state
           if (!self.mockHistory[repositoryName]) {
             self.mockHistory = { ...self.mockHistory, [repositoryName]: {} };
           }
+
+          // is this the same issue as above, overwriting the existing history when a new method is passed?
           self.mockHistory[repositoryName] = { [method]: 0 };
```

Comment:
Yes you are right, we also need to do the same that you just did above in the code. I don't know what I was thinking but thank you to highlight this bug. Appreciate it man. Can you please correct this as well. 
And then I'll merge the changes to master.

Reply:
I believe this issue is resolved now

Reply:
No problem, thanks for the library 🥇 
I believe this issue is resolved hopefully 👍